R/statistic-histogram.R

```diff
@@ -82,10 +82,10 @@ dpHistogram$methods(
         
         # get the epsilon and accuracy
         if (is.null(epsilon)) {
-            .self$accuracy <- accuracy
+            .self$accuracy <- checkAccuracy(accuracy)
             .self$epsilon <- histogramGetEpsilon(mechanism, accuracy, delta, alpha, .self$sens)
         } else {
-            .self$epsilon <- epsilon
+            .self$epsilon <- checkEpsilon(epsilon)
```

Comment:
why does this checkEpsilon() call not include "expectedLength = 1"?

Reply:
because expectedLength defaults to 1.